plugins/flutter_plugins/shared_preferences_fde/macos/Classes/SharedPreferencesPlugin.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import FlutterMacOS
+import Foundation
+
+public class SharedPreferencesPlugin: NSObject, FlutterPlugin {
+  public static func register(with registrar: FlutterPluginRegistrar) {
+    let channel = FlutterMethodChannel(
+      name: "plugins.flutter.io/shared_preferences",
```

Comment:
Should these strings (and below) be extracted to constants?

Reply:
I followed the flutter/plugins style for these, which at least in the plugins I was looking at tended to inline them. I think for simple plugins like this it doesn't matter much one way or the other, which is why I opted for that style rather than what we've been doing with the FDE plugins. When this moves to an official location I think we should have a written style on this.